{
	"id": "CVE-2019-16935",
	"vulnerabilities": [
		{
			"id": "CVE-2019-16935",
			"references": [
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.alpinelinux.org/vuln/CVE-2019-16935"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "alpine:3.12",
			"vulnerable": true,
			"package": {
				"name": "python2",
				"repositories": [
					"main"
				],
				"architectures": [
					"aarch64",
					"armhf",
					"armv7",
					"mips64",
					"ppc64le",
					"s390x",
					"x86",
					"x86_64"
				]
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "2.7.16-r3"
					}
				],
				"fixed": [
					"2.7.16-r3"
				]
			}
		},
		{
			"ecosystem": "alpine:3.12",
			"vulnerable": true,
			"package": {
				"name": "python3",
				"repositories": [
					"main"
				],
				"architectures": [
					"aarch64",
					"armhf",
					"armv7",
					"mips64",
					"ppc64le",
					"s390x",
					"x86",
					"x86_64"
				]
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "3.7.5-r0"
					}
				],
				"fixed": [
					"3.7.5-r0"
				]
			}
		}
	],
	"data_source": "alpine-secdb"
}
